main/trello/api/lists_manager.py

```diff
@@ -0,0 +1,60 @@
+from urllib import response
+from main.trello.api.rest_base_manager import RESTBaseManager
+
+"""
+Module to manage Trello Lists through the API
+
+Classes:
+    ListsManager
+"""
+class ListsManager(RESTBaseManager):
+    """Class which can be used to manage Trello Lists through the API"""
+
+    def __init__(self, request_method=None):
+        super().__init__(request_method)
+        """ Construct the necessary attributes for the ListsManager
+
+        :param method: obj  RequestManager object which is used to handle the API requests
+        """
+    
+    def create_list(self, name, idBoard):
+        """ Create a List in a Board
+
+        :param name: str   Name of the List
+        :param idBoard:  str   ID of the Board to which the List belongs
+        :return: Tuple that contains the status code and the response.
+        """
+        endpoint = f"lists/?name={name}&idBoard={idBoard}"
+        payload = { "name": name}
```

Comment:
Probably a nitpick, but sending a parameter ('name') both as a query parameter AND inside the payload seems redundant and unnecessary.

Reply:
Resolved with the changes suggested.